### DISCUSSION TITLE
"mail.sig" in Jakarta Mail misclassified as Standard ML

### DISCUSSION THREAD

**Discussion by ShreckYe** [github-linguist/linguist#5691] [Classification] [answered]
[The "mail.sig" file in Jakarta Mail](https://github.com/eclipse-ee4j/mail/blob/2fe6c37d633e32f798705fb47875dca8987fbb28/mail.sig) is misclassified as Standard ML.

**Comment by lildude** [comment, marked as answer]
That will be because `.sig` is only associated with Standard ML and no other language. If this is a common extension associated with another language, feel free to submit a PR to add support. Until then, you'll need to implement an override to force the language of you choice.
